client/src/main/scala/com/dwolla/cloudflare/domain/model/accounts/AccountMember.scala

```diff
@@ -2,6 +2,8 @@ package com.dwolla.cloudflare.domain.model.accounts
 
 case class AccountMember (
   id: String,
+  accountId: String,
+  uri: String,
```

Comment:
Instead of having a `uri` field, could we have a function somewhere to convert `AccountMember` to URI? Or `accountId ⇒ AccountMember ⇒ URI`?

Reply:
We could add a function to convert AccountMember to URI, but it would need an account id. That is probably ok.